rfc8446.xml

```diff
@@ -2262,7 +2263,7 @@ extension (see <xref target="early-data-indication"/>), the first identity is th
 one used for 0-RTT data.</t>
   <t hangText='binders:'>
   A series of HMAC values, one for
-each PSK offered in the "pre_shared_keys" extension and in the same
+each PSK identity in identities and in the same
```

Comment:
hmm.... I think "each offered PSK". "in identities" seems weird.

Reply:
The key thing I noted here is that we're describing the fields of the "pre_shared_keys" extension, and the self-reference is pretty weird/unhelpful.

Reply:
Rewritten a bit.